### DISCUSSION TITLE
Failed to extract pimcore/data-hub-simple-rest and Failed to extract pimcore/openid-connect

### DISCUSSION THREAD

**Discussion by BjBue** [pimcore/pimcore#15261] [Q&A / Support] [answered]
Hello folks,

this is what the

`COMPOSER_MEMORY_LIMIT=-1 composer install --ignore-platform-reqs --no-scripts --optimize-autoloader --apcu-autoloader `

gives me as answer:

` Failed to extract pimcore/data-hub-simple-rest: (9) '/usr/bin/unzip' -qq '/php/vendor/composer/tmp-0eb6859d42f49c07c34e795fe999
f199' -d '/php/vendor/composer/8730c954'
#0 336.9
#0 336.9 [/php/vendor/composer/tmp-0eb6859d42f49c07c34e795fe999f199]
#0 336.9   End-of-central-directory signature not found.  Either this file is not
#0 336.9   a zipfile, or it constitutes one disk of a multi-part archive.  In the
#0 336.9   latter case the central directory and zipfile comment will be found on
#0 336.9   the last disk(s) of this archive.
#0 336.9 unzip:  cannot find zipfile directory in one of /php/vendor/composer/tmp-0eb6859d42f49c07c34e795fe999f199 or
#0 336.9         /php/vendor/composer/tmp-0eb6859d42f49c07c34e795fe999f199.zip, and cannot find /php/vendor/composer/tmp-0eb6859d42f49c07c34e795fe999f199.ZIP, period.
#0 336.9
#0 336.9     The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
#0 336.9     Unzip with unzip command failed, falling back to ZipArchive class
#0 336.9     Install of pimcore/data-hub-simple-rest failed
`
and the next is:
`#0 337.0  248/266 [==========================>-]  93%    Failed to extract pimcore/openid-connect: (9) '/usr/bin/unzip' -qq '/php/vendor/composer/tmp-d53f1e415b096de8acca95b8a03a0703' 
-d '/php/vendor/composer/f6126d8c'
#0 337.1
#0 337.1 [/php/vendor/composer/tmp-d53f1e415b096de8acca95b8a03a0703]
#0 337.1   End-of-central-directory signature not found.  Either this file is not
#0 337.1   a zipfile, or it constitutes one disk of a multi-part archive.  In the
#0 337.1   latter case the central directory and zipfile comment will be found on
#0 337.1   the last disk(s) of this archive.
#0 337.1 unzip:  cannot find zipfile directory in one of /php/vendor/composer/tmp-d53f1e415b096de8acca95b8a03a0703 or
#0 337.1         /php/vendor/composer/tmp-d53f1e415b096de8acca95b8a03a0703.zip, and cannot find /php/vendor/composer/tmp-d53f1e415b096de8acca95b8a03a0703.ZIP, period.
#0 337.1
#0 337.1     The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
#0 337.1     Unzip with unzip command failed, falling back to ZipArchive class
#0 337.1     Install of pimcore/openid-connect failed
#0 337.1     Failed to extract pimcore/statistics-explorer: (9) '/usr/bin/unzip' -qq '/php/vendor/composer/tmp-8fe83bf10b628c7630041914cfc54cc5' -d '/php/vendor/composer/eea0b04e'     
#0 337.1
#0 337.1 [/php/vendor/composer/tmp-8fe83bf10b628c7630041914cfc54cc5]
#0 337.1   End-of-central-directory signature not found.  Either this file is not
#0 337.1   a zipfile, or it constitutes one disk of a multi-part archive.  In the
#0 337.1   latter case the central directory and zipfile comment will be found on
#0 337.1   the last disk(s) of this archive.
#0 337.1 unzip:  cannot find zipfile directory in one of /php/vendor/composer/tmp-8fe83bf10b628c7630041914cfc54cc5 or
#0 337.1         /php/vendor/composer/tmp-8fe83bf10b628c7630041914cfc54cc5.zip, and cannot find /php/vendor/composer/tmp-8fe83bf10b628c7630041914cfc54cc5.ZIP, period.
#0 337.1
#0 337.1     The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
#0 337.1     Unzip with unzip command failed, falling back to ZipArchive class
#0 337.1     Install of pimcore/statistics-explorer failed
#0 337.1     Failed to extract pimcore/asset-metadata-class-definitions: (9) '/usr/bin/unzip' -qq '/php/vendor/composer/tmp-eeb88e00b6f311bdfa05284e204c1889' -d '/php/vendor/composer/9
3d4d433'
#0 337.1
#0 337.1 [/php/vendor/composer/tmp-eeb88e00b6f311bdfa05284e204c1889]
#0 337.1   End-of-central-directory signature not found.  Either this file is not
#0 337.1   a zipfile, or it constitutes one disk of a multi-part archive.  In the
#0 337.1   latter case the central directory and zipfile comment will be found on
#0 337.1   the last disk(s) of this archive.
#0 337.1 unzip:  cannot find zipfile directory in one of /php/vendor/composer/tmp-eeb88e00b6f311bdfa05284e204c1889 or
#0 337.1         /php/vendor/composer/tmp-eeb88e00b6f311bdfa05284e204c1889.zip, and cannot find /php/vendor/composer/tmp-eeb88e00b6f311bdfa05284e204c1889.ZIP, period.
#0 337.1
#0 337.1     The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
#0 337.1     Unzip with unzip command failed, falling back to ZipArchive class
#0 337.1     Install of pimcore/asset-metadata-class-definitions failed
#0 337.2     Failed to extract pimcore/portal-engine: (9) '/usr/bin/unzip' -qq '/php/vendor/composer/tmp-c8becd2451c6a92379260c96bf9ed548' -d '/php/vendor/composer/c500c931'
#0 337.2
#0 337.2 [/php/vendor/composer/tmp-c8becd2451c6a92379260c96bf9ed548]
#0 337.2   End-of-central-directory signature not found.  Either this file is not
#0 337.2   a zipfile, or it constitutes one disk of a multi-part archive.  In the
#0 337.2   latter case the central directory and zipfile comment will be found on
#0 337.2   the last disk(s) of this archive.
#0 337.2 unzip:  cannot find zipfile directory in one of /php/vendor/composer/tmp-c8becd2451c6a92379260c96bf9ed548 or
#0 337.2         /php/vendor/composer/tmp-c8becd2451c6a92379260c96bf9ed548.zip, and cannot find /php/vendor/composer/tmp-c8becd2451c6a92379260c96bf9ed548.ZIP, period.
#0 337.2
#0 337.2     The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
#0 337.2     Unzip with unzip command failed, falling back to ZipArchive class
#0 337.2     Install of pimcore/portal-engine failed
`
Did anybody have a n idea?

**Comment by NiklasBr** [comment]
I think there is some general error, having trouble updating myself:

```
composer outdated -v

In JsonFile.php line 347:

  [Seld\JsonLint\ParsingException]
  "https://enterprise.repo.pimcore.com/packages.json" does not contain valid JSON
  Parse error on line 1:
  ...ject-search":"^3.1","pimc
  ----------------------^
  Invalid string, it appears you forgot to terminate a string, or attempted to write a multiline string which is invalid - It appears you have an extra trailing comma


Exception trace:
  at phar:///usr/bin/composer/src/Composer/Json/JsonFile.php:347
 Composer\Json\JsonFile::validateSyntax() at phar:///usr/bin/composer/src/Composer/Json/JsonFile.php:321
 Composer\Json\JsonFile::parseJson() at phar:///usr/bin/composer/src/Composer/Util/Http/Response.php:96
 Composer\Util\Http\Response->decodeJson() at phar:///usr/bin/composer/src/Composer/Repository/ComposerRepository.php:1479
 Composer\Repository\ComposerRepository->fetchFile() at phar:///usr/bin/composer/src/Composer/Repository/ComposerRepository.php:1169
 Composer\Repository\ComposerRepository->loadRootServerFile() at phar:///usr/bin/composer/src/Composer/Repository/ComposerRepository.php:800
 Composer\Repository\ComposerRepository->hasProviders() at phar:///usr/bin/composer/src/Composer/Repository/ComposerRepository.php:479
 Composer\Repository\ComposerRepository->loadPackages() at phar:///usr/bin/composer/src/Composer/Repository/RepositorySet.php:197
 Composer\Repository\RepositorySet->findPackages() at phar:///usr/bin/composer/src/Composer/Package/Version/VersionSelector.php:86
 Composer\Package\Version\VersionSelector->findBestCandidate() at phar:///usr/bin/composer/src/Composer/Command/ShowCommand.php:1399
 Composer\Command\ShowCommand->findLatestPackage() at phar:///usr/bin/composer/src/Composer/Command/ShowCommand.php:449
 Composer\Command\ShowCommand->execute() at phar:///usr/bin/composer/vendor/symfony/console/Command/Command.php:298
 Symfony\Component\Console\Command\Command->run() at phar:///usr/bin/composer/vendor/symfony/console/Application.php:1040
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/bin/composer/vendor/symfony/console/Application.php:301
 Symfony\Component\Console\Application->doRun() at phar:///usr/bin/composer/src/Composer/Console/Application.php:377
 Composer\Console\Application->doRun() at phar:///usr/bin/composer/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at phar:///usr/bin/composer/src/Composer/Console/Application.php:141
 Composer\Console\Application->run() at phar:///usr/bin/composer/src/Composer/Command/OutdatedCommand.php:109
 Composer\Command\OutdatedCommand->execute() at phar:///usr/bin/composer/vendor/symfony/console/Command/Command.php:298
 Symfony\Component\Console\Command\Command->run() at phar:///usr/bin/composer/vendor/symfony/console/Application.php:1040
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/bin/composer/vendor/symfony/console/Application.php:301
 Symfony\Component\Console\Application->doRun() at phar:///usr/bin/composer/src/Composer/Console/Application.php:377
 Composer\Console\Application->doRun() at phar:///usr/bin/composer/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at phar:///usr/bin/composer/src/Composer/Console/Application.php:141
 Composer\Console\Application->run() at phar:///usr/bin/composer/bin/composer:88
 require() at /usr/bin/composer:29
```

**Comment by fashxp** [comment, marked as answer]
That's a repman problem, see also recent issues here: https://github.com/repman-io/repman/issues

Right now, we cannot do anything other than wait 😒. 
We are working on moving away from it though... 

Sorry for any inconvenience.

**Reply by BjBue** [reply]
Thanks for the info! Good luck with all the work that needs to be done ;-)
